_docs/collaboration.md

```diff
@@ -17,12 +17,33 @@ For communication in between your team, you can look into the following:
 
 ## Collaborative Coding
 To keep track of your code and prevent issues utilize the following:
--[Git](git-scm.com) = Track changes and sync your code with others
--[GitHub](Github.com) = Keep all your coding files in sync and help track changes in a Public or Private repo.
+- [Git](git-scm.com) = Track changes and sync your code with others
+- [GitHub](Github.com) = Keep all your coding files in sync and help track changes in a Public or Private repo.
 *Note:* Github has something called "Projects" which is very much like Trello boards. So you can take advantage of these features!
```

Comment:
Using spaces to keep line spacing for bulleted items consistent (unsure if it'll matter too much but it's a good idea anyhow)
```suggestion
  *Note:* Github has something called "Projects" which is very much like Trello boards. So you can take advantage of these features!
```